Merge Intervals.cpp

```diff
@@ -0,0 +1,22 @@
+class Solution {
+public:
+    vector<vector<int>> merge(vector<vector<int>>& intervals) {
```

Comment:
Instead of a class based solution please can you provide a function/sequential solution so that beginners(with no knowledge of OOPD) won't get trouble understanding the solution.